sdk/RELEASE.md

```diff
@@ -46,6 +46,7 @@
 * Load v1 and v2 component yaml into v2 ComponentSpec and convert v1 component
   spec to v2 component spec [\#6497](https://github.com/kubeflow/pipelines/issues/6497)
 * Format all Python files under SDK folder. [\#6501](https://github.com/kubeflow/pipelines/issues/6501)
+* Get short name of complex input/output types to ensure we can map to appropriate de|serializer. [\#6504](https://github.com/kubeflow/pipelines/pull/6504)
```

Comment:
This is under `1.8.0` which is already released. Can you please move it up to `Current Version (Still in Development)`? Otherwise LGTM.

Reply:
🤦  Thanks for spotting that! Fixed.